### PR TITLE
add warning about ephemeral db storage

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@adellape ptal.
@smarterclayton fyi.

File: dev_guide/templates.adoc [hunk 1]
```diff
@@ -236,3 +236,8 @@ from the template, specifying your fork instead of the default value.
 By doing this, the build configuration created by the template will now point to
 your fork of the application code, and you can modify the code and rebuild the
 application at will.
+
+[NOTE]
+====
+Some of the quickstart templates define a database deployment configuration.  The configuration they define uses ephemeral storage for the database content.  These templates should be used for demonstration purposes only as all database data will be lost if the database pod restarts for any reason.
```
Comment (adellape) [inline dev_guide/templates.adoc]: s/quickstart/QuickStart/

**Comment by bparees** [comment]
fixed

**Comment by adellape** [comment]
LGTM

**Pull request merged**